app/payments/get-data.js

```diff
@@ -0,0 +1,21 @@
+const { v4: uuidv4 } = require('uuid')
+const { sendMessage, receiveMessage } = require('../messaging')
+const config = require('../config')
+const util = require('util')
+const { TYPE } = require('../constants/type')
```

Comment:
In case my links don't work this time, I'll just this out again but -

Would it be better to put these in the order they appear in the file?

Reply:
Or if not, I saw you had put the constants at the top of the file in a few other files below, so maybe it would be better to do that to stay consistent?

Reply:
Updated